src/actions/continuous-deploy-fingerprint.ts

```diff
@@ -30,63 +33,103 @@ export async function continuousDeployFingerprintAction(input = collectContinuou
     return setFailed(`Missing 'extra.eas.projectId' in app.json or app.config.js.`);
   }
 
-  const androidFingerprintHash = await getFingerprintHashForPlatformAsync({
-    cwd: input.workingDirectory,
-    platform: 'android',
-  });
-  const iosFingerprintHash = await getFingerprintHashForPlatformAsync({ cwd: input.workingDirectory, platform: 'ios' });
+  let platform: PlatformArg = 'all';
+  switch (input.platform) {
+    case 'android':
+      platform = 'android';
+      break;
+    case 'ios':
+      platform = 'ios';
+      break;
+  }
 
-  info(`Android fingerprint: ${androidFingerprintHash}`);
-  info(`iOS fingerprint: ${iosFingerprintHash}`);
+  async function buildAndroid() {
+    if (platform === 'ios') return;
```

Comment:
small nit (null is clearer in result type than void):
```suggestion
    if (platform === 'ios') {
      return null;
    }
```